docs/docsite/rst/dev_guide/developing_modules_in_groups.rst

```diff
@@ -1,64 +1,49 @@
-* Shared documentation (for example describing common arguments) can be placed in ``lib/ansible/plugins/doc_fragments/``.
-* With great power comes great responsibility: Ansible module maintainers have a duty to help keep modules up to date. As with all successful community projects, module maintainers should keep a watchful eye for reported issues and contributions.
-* Although not required, unit and/or integration tests are strongly recommended. Unit tests are especially valuable when external resources (such as cloud or network devices) are required. For more information see :ref:`developing_testing` and the `Testing Working Group <https://github.com/ansible/community/blob/master/meetings/README.md>`_.
-  * Starting with Ansible 2.4 all :ref:`network_modules` MUST have unit tests.
+* We encourage PEP 8 compliance. See :ref:`testing_pep8` for more information.
+* We encourage supporting :ref:`Python 2.6+ and Python 3.5+ <developing_python_3>`.
+* Look at Ansible Galaxy and review the naming conventions in your functional area (such as cloud, networking, databases).
+* With great power comes great responsibility: Ansible collection maintainers have a duty to help keep content up to date. As with all successful community projects, collection maintainers should keep a watchful eye for reported issues and contributions.
+* We strongly recommend unit and/or integration tests. Unit tests are especially valuable when external resources (such as cloud or network devices) are required. For more information see :ref:`developing_testing` and the `Testing Working Group <https://github.com/ansible/community/blob/master/meetings/README.md>`_.
 
 
-Naming convention
-=================
+Naming conventions
+==================
 
-As you may have noticed when looking under ``lib/ansible/modules/`` we support up to two directories deep (but no deeper), e.g. `databases/mysql`. This is used to group files on disk as well as group related modules into categories and topics the Module Index, for example: :ref:`database_modules`.
+Fully Qualified Collection Names (FQCNs) for plugins and modules include three elements:
 
-The directory name should represent the *product* or *OS* name, not the company name.
+  * the Galaxy namespace, which generally represents the company or group
+  * the collection name, which generally represents the product or OS
+  * the plugin or module name
+      * always in lower case
+      * words separated with an underscore (``_``) character
+      * singular, rather than plural, for example, ``command`` not ``commands``
 
-Each module should have the above (or similar) prefix; see existing :ref:`all_modules` for existing examples.
-
-**Note:**
-
-* File and directory names are always in lower case
-* Words are separated with an underscore (``_``) character
-* Module names should be in the singular, rather than plural, eg ``command`` not ``commands``
+For example, ``community.mongodb.mongdb_linux`` or ``cisco.meraki.meraki_device``.
```

Comment:
`mongdb` is intentional ?

Reply:
Nope, that's a typo. Thanks @bmillemathias!